src/views/pages/models.jsx

```diff
@@ -19,10 +18,14 @@ export default class Models extends ListenerComponent {
             models: ModelStore.getAllModels()
         };
     }
+    addNewModel(){
+        hashHistory.push("/models/new");
+    }
     render() {
         return (
             <div id="models">
                 <h1>All models</h1>
+                <button onClick={this.addNewModel}>Make new model</button>
```

Comment:
Nit: This would be a good use case for `import { Link } from 'react-router'` and `<Link to='/models/new'>Make new model</Link>`.